source/frontend/package.json

```diff
@@ -29,7 +29,7 @@
     "is-absolute-url": "3.0.3",
     "js-file-download": "0.4.12",
     "keycloak-js": "26.0.0",
-    "leaflet": "1.9.2",
+    "leaflet": "1.9.4",
```

Comment:
I think this is fine but any change in dependencies might introduce further errors/regressions. Would prefer to do this outside of horfixes (unless the fix is to update the library...)